config/application.rb

```diff
@@ -146,6 +146,9 @@ class Application < Rails::Application
         child.helper helpers
       end
     end
+
+    # Should ANSI color codes be used when logging information
+    config.colorize_logging = SETTINGS[:colorize_logging]
```

Comment:
I think this makes sense to have different values per environment. We should have it enabled by default for development and disabled for test and production environments. Instead of introducing 3 new settings, I suggest we should make only production configurable.